pkg/plugin/plugin.go

```diff
@@ -116,15 +117,15 @@ func generateRandomMac() net.HardwareAddr {
 	return net.HardwareAddr(append(prefix, suffix...))
 }
 
-func setupVeth(contNetns ns.NetNS, contIfaceName string, requestedMac string) (*current.Interface, *current.Interface, error) {
+func setupVeth(contNetns ns.NetNS, contIfaceName string, requestedMac string, mtu int) (*current.Interface, *current.Interface, error) {
 	hostIface := &current.Interface{}
 	contIface := &current.Interface{}
 
 	// Enter container network namespace and create veth pair inside. Doing
 	// this we will make sure that both ends of the veth pair will be removed
 	// when the container is gone.
 	err := contNetns.Do(func(hostNetns ns.NetNS) error {
-		hostVeth, containerVeth, err := ip.SetupVeth(contIfaceName, 1500, hostNetns)
+		hostVeth, containerVeth, err := ip.SetupVeth(contIfaceName, mtu, hostNetns)
```

Comment:
what if the MTU is not specified? do we set it to 0?

Reply:
when it's 0 the kernel allocate a default value.

added a check for that